website/docs/d/organization_members.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization_members"
+sidebar_current: "docs-datasource-tfe-organization-members"
+description: |-
+  Get information on an Organization members.
+---
+
+# Data Source: tfe_organization_members
+
+Use this data source to get information about members of an organization.
+
+## Example Usage
+
+```hcl
+data "tfe_organization_members" "foo" {
+  organization = "organization-name"
+}
```

Comment:
💅 Typically in our examples we also include defining the resources our documented resource would depend on:

```hcl
resource "tfe_organization" "deathstar" {
   name = "deathstar"
   email = "sheev.palpatine@hashicorp.com"
}

data "tfe_organization_members" "stormtroopers" {
  organization = tfe_organization.deathstar.name
}
```
😄

Reply:
good point!